eap74-rhel8-payg-multivm/src/main/scripts/enable-elytron-se17-domain.cli

```diff
@@ -0,0 +1,446 @@
+# This CLI script modifies the default domain configuration files (domain.xml,
+# host.xml, host-master.xml and host-slave.xml) in order to remove legacy
+# security, configure elytron and be able to be run the domain with Java SE 17.
+# There are different sections to prepare the profiles, remove legacy extensions
+# and modify the host controller files. The file can be used as an example to
+# prepare other domain configuration files for Java 17.
+#
```

Comment:
Hello @zhengchang907 , can you please help me understand why we need to include the `enable-elytron-se17-domain.cli` here in the payg-multivm offer, but in the vmss and single node offers we do not?

Reply:
Hi @edburns, I added comments to the `enable-elytron-se17-domain.cli` file as we discussed. This is a temporary solution to: https://issues.redhat.com/browse/JBEAP-23983